examples/svm/plot_separating_hyperplane.py

```diff
@@ -12,12 +12,13 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from sklearn import svm
-
+from sklearn.datasets import make_classification
```

Comment:
@lesteve under the Issue  #8364 , @amueller point out it good to use `make_classification` for data generation.